omise/test/__init__.py

```diff
@@ -1,3952 +1,10 @@
 import sys
-import unittest
-import mock
-
-
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
 
 try:
     basestring
 except NameError:
     basestring = str
 
-
 if sys.version_info[0] == 2:
```

Comment:
I can't recalled why this 2.x condition is needed (Python 2 has `next`), and since test seems to pass without it, can you remove (along with `import sys`)?